webpack.base.js

```diff
@@ -9,6 +9,13 @@ exports.moduleConfig = {
       test: /\.vue$/,
       exclude: /node_modules/,
       loader: 'vue-loader'
+    },
+    {
+      test: /\.css$/,
+      loader: [
```

Comment:
to load [main.css](https://github.com/uncovertruth/userdive-tag-paster/blob/master/app/styles/main.css)